### ISSUE TITLE
Weekly Report - Improve Naming on Weekly report email

### ISSUE THREAD

**Issue by Angelodaniel** [getsentry/sentry#63190]
### Problem Statement

Currently when the weekly report gets in we show you the Errors accepted / dropped and Transactions accepted / dropped.
The naming dropped implies that you are dropping a lot while it actually displays Dropped + Filtered.

This can cause some unwanted reactions and causes the user to dive deeper into the numbers while everything is under control already

### Solution Brainstorm

Improve naming or split the dropped column into filtered + dropped.

### Product Area

Unknown

**Comment by hubertdeng123** [comment]
I'm not sure exactly where this should go, but looks like weekly reports is under the `Settings`? Let me know if this should be rerouted

**Comment by isabellaenriquez** [comment]
Yeah I agree that this is pretty confusing. Will add to the backlog, thank you for pointing that out!
